tactics/tacticals.ml

```diff
@@ -434,9 +434,13 @@ module New = struct
     tclIFTHENELSE t1 (tclTRY t2) t2
 
   (* Try the first tactic that does not fail in a list of tactics *)
-  let rec tclFIRST = function
-    | [] -> tclZEROMSG (str"No applicable tactic.")
-    |  t::rest -> tclORELSE0 t (tclFIRST rest)
+  let rec tclFIRST ?(msg = str "No applicable tactic.") = function
+    | [] -> tclZEROMSG msg
+    | t::rest -> tclORELSE0 t (tclFIRST ~msg rest)
+
+  let rec tclMULTIFIRST ?(msg = str "No applicable tactic.") = function
```

Comment:
There is already a `tclANY` primitive in `tactics.ml` that does the same thing, by the way...